### PR TITLE
Simplify bootstrapping with dune

### PR DESCRIPTION
**Pull request by leostera**
Hi folks 👋🏼 just got around this page and thought it could be simplified a little. Feel free to dismiss if it doesn't meet the expected guidelines/style. It includes:

* simpler, more direct language (no "Thus", less adverbs, address the reader directly)
* using a more recognizable shell syntax
* dropping the `path` parameter that I haven't seen folks use before
* don't mention _kinds_ (which should really be called _components_ anyway) 
* give very straightforward cases of creating a new library, test, and executable

### CODE REVIEW

File: data/tutorials/platform/bp_00_bootstrap_project.md [hunk 1]
```diff
@@ -8,58 +8,47 @@ category: "Best Practices"
 
 # Bootstrapping a Project with Dune
 
-[Dune](https://dune.readthedocs.io/en/stable/overview.html) is recommended for bootstrapping projects using `dune init`. If `opam` or `dune` are not installed, please see [the OCaml install page](/install).
+[Dune](https://dune.readthedocs.io/en/stable/overview.html) is recommended for bootstrapping projects using `dune init`. To install `dune`, please see [the OCaml install page](/install).
```
Comment (cuihtlauac) [inline data/tutorials/platform/bp_00_bootstrap_project.md]:
```suggestion
[Dune](https://dune.readthedocs.io/en/stable/overview.html) is recommended for bootstrapping projects. To install `dune`, please see [the OCaml install page](/install).
```

**Review by cuihtlauac — APPROVED** [review]

**Comment by cuihtlauac** [comment]
This is failing on unrelated omd to cmarkit issues: 

https://deploy.ci.ocaml.org/job/2023-11-24/123654-ocluster-build-2542d1

Is this a transient problem?

**Comment by sabine** [comment]
Rebased on main after merge of https://github.com/ocaml/ocaml.org/pull/1642 and force-pushed to see if CI is fine now

**Comment by cuihtlauac** [comment]
> Rebased on main after merge of https://github.com/ocaml/ocaml.org/pull/1642 and force-pushed to see if CI is fine now

Usually, it's not needed. Re-running the tests is sufficient. However, you have to wait about half an hour for the build fix PR to become active.

**Pull request merged**